pkg/planner/core/explain.go

```diff
@@ -270,6 +271,24 @@ func (p *PhysicalTableScan) OperatorInfo(normalized bool) string {
 			buffer.WriteString(runtimeFilter.ExplainInfo(false))
 		}
 	}
+	if p.AnnIndexExtra != nil && p.AnnIndexExtra.ANNQueryInfo.TopK != math.MaxUint32 {
```

Comment:
When `ANNQueryInfo.TopK != math.MaxUint32` is true? [Here](https://github.com/pingcap/tidb/pull/56204/files#diff-7ab669fbd07f777ce6a8b33025e95960e50d6c6be00a0781d33aafca2150bf6bR2489) we set it to `math.MaxUint32`